llvm/lib/IR/RepoTicket.cpp

```diff
@@ -116,7 +116,39 @@ static GODigestState accumulateGOFinalDigestOrInitialDigestAndGetGODigestState(
   return accumulateGOInitialDigestAndGetGODigestState(GO, GOHash, GOIMap);
 }
 
-// Update the GO's hash value by adding the hash of its dependents.
+// Loop through the Contributions and add the the G's initial digest to the
+// GOHash. Return true if the GOhash is changed.
+static bool updateDigestUseContributions(MD5 &GOHash,
+                                         const GOVec &Contributions,
+                                         GOInfoMap &GOIMap) {
+  for (const GlobalObject *const G : Contributions) {
+    GOHash.update(GOIMap[G].InitialDigest.Bytes);
+  }
+  return !Contributions.empty();
+}
+
+// Loop through the Dependencies and add the the G's digest to the GOHash.
```

Comment:
This comment needs to be tweaked. 
- "Loop through the dependences" (dependencies isn't a proper noun). 
- Note sure what "the the G's digest" is. 
- "to the GOHash" -> "to GOHash".

Reply:
Thanks, I have updated the comment.